app/concepts/todo/create.rb

```diff
@@ -1,50 +1,33 @@
 class Todo::Create < Trailblazer::Operation
+  include Model
+  model Todo, :create
 
-  # Logic for creating a TODO
-  #     if `todo_list` is specified
-  #         create the `todo`
-  #         associated with `todo_list`
-  #     else
-  #         create the `todo`
-  #         create a `user`
-  #         create a default `todo_list`
-  #         associate `todo_list` with `user`
-  #         associate `todo` with `todo_list`
-  def process(params)
-    # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
-
-    # build the todo
-    todo = @model = Todo.new(params.require(:todo).permit(:title, :description))
+  contract do
+    property :title, validates: {presence: true}
+  end
 
-    # find or create the parent list for the todo
-    if params[:todo_list_id].blank?
-      todo_list = TodoList.find_or_create_by(name: "Default To-do List")
+  def process(params)
+    tlid, cuid = params[:todo_list_id], params[:current_user_id]
```

Comment:
Hi @erbunao, does it make sense to do this `before` or `after` the todo params have been validated?